pyright", "credits" or "license" for more information.
>>> import essentia, essentia.streaming, essentia.standard
[ WARNING  ] ARGLLLLLL
[ WARNING  ] ARGLLLLLL
>>> loader = essentia.standard.MonoLoader(filename = '~/documents/hellobrooklyn.mp3')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/local/lib/python2.7/dist-packages/essentia/standard.py", line 43, in __init__
    self.configure(**kwargs)
  File "/usr/local/lib/python2.7/dist-packages/essentia/standard.py", line 58, in configure
    self.__configure__(**kwargs)
RuntimeError: Error while configuring MonoLoader: AudioLoader: Could not open file "~/documents/hellobrooklyn.mp3", error = No such file or directory
>>> loader = essentia.standard.MonoLoader(filename = '~/Documents/hellobrooklyn.mp3')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/local/lib/python2.7/dist-packages/essentia/standard.py", line 43, in __init__
    self.configure(**kwargs)
  File "/usr/local/lib/python2.7/dist-packages/essentia/standard.py", line 58, in configure
    self.__configure__(**kwargs)
RuntimeError: Error while configuring MonoLoader: AudioLoader: Could not open file "~/Documents/hellobrooklyn.mp3", error = No such file or directory
>>> loader = essentia.standard.MonoLoader(filename = '/home/Documents/hellobrooklyn.mp3')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/local/lib/python2.7/dist-packages/essentia/standard.py", line 43, in __init__
    self.configure(**kwargs)
  File "/usr/local/lib/python2.7/dist-packages/essentia/standard.py", line 58, in configure
    self.__configure__(**kwargs)
RuntimeError: Error while configuring MonoLoader: AudioLoader: Could not open file "/home/Documents/hellobrooklyn.mp3", error = No such file or directory
>>> loader = essentia.standard.MonoLoader(filename = './home/Documents/hellobrooklyn.mp3')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/local/lib/python2.7/dist-packages/essentia/standard.py", line 43, in __init__
    self.configure(**kwargs)
  File "/usr/local/lib/python2.7/dist-packages/essentia/standard.py", line 58, in configure
    self.__configure__(**kwargs)
RuntimeError: Error while configuring MonoLoader: AudioLoader: Could not open file "./home/Documents/hellobrooklyn.mp3", error = No such file or directory
>>> loader = essentia.standard.MonoLoader(filename = 'home/Documents/hellobrooklyn.mp3')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/local/lib/python2.7/dist-packages/essentia/standard.py", line 43, in __init__
    self.configure(**kwargs)
  File "/usr/local/lib/python2.7/dist-packages/essentia/standard.py", line 58, in configure
    self.__configure__(**kwargs)
RuntimeError: Error while configuring MonoLoader: AudioLoader: Could not open file "home/Documents/hellobrooklyn.mp3", error = No such file or directory
>>> loader = essentia.standard.MonoLoader(filename = '/home/bzachmann/Documents/hellobrooklyn.mp3')
>>> audio = loader()
[ WARNING  ] AudioLoader: invalid frame, skipping it: Invalid data found when processing input
>>> from pylab import plot, show, figure
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ImportError: No module named pylab
>>> from pylab import plot, show, figure
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ImportError: No module named pylab
>>> from pylab import plot, show, figure
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ImportError: No module named pylab
>>> danceability = essentia.standard.danceablility(audio)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: 'module' object has no attribute 'danceablility'
>>> danceability = essentia.standard.Danceablility(audio)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: 'module' object has no attribute 'Danceablility'
>>> danceability = essentia.standard.Danceability(audio)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: __init__() takes exactly 1 argument (2 given)
>>> danceability = essentia.standard.Danceablility(file = audio)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: 'module' object has no attribute 'Danceablility'
>>> danceability = essentia.standard.Danceability(file = audio)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/local/lib/python2.7/dist-packages/essentia/standard.py", line 43, in __init__
    self.configure(**kwargs)
  File "/usr/local/lib/python2.7/dist-packages/essentia/standard.py", line 48, in configure
    goalType = self.paramType(name)
ValueError: 'file' is not a parameter of Danceability
>>> danceability = essentia.standard.Danceability(input  = audio)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/local/lib/python2.7/dist-packages/essentia/standard.py", line 43, in __init__
    self.configure(**kwargs)
  File "/usr/local/lib/python2.7/dist-packages/essentia/standard.py", line 48, in configure
    goalType = self.paramType(name)
ValueError: 'input' is not a parameter of Danceability
>>> danceability = essentia.standard.Danceability(audio)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: __init__() takes exactly 1 argument (2 given)
>>> danceability = essentia.standard.Danceability(filename = audio)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/local/lib/python2.7/dist-packages/essentia/standard.py", line 43, in __init__
    self.configure(**kwargs)
  File "/usr/local/lib/python2.7/dist-packages/essentia/standard.py", line 48, in configure
    goalType = self.paramType(name)
ValueError: 'filename' is not a parameter of Danceability
>>> danceability = essentia.standard.Danceability()
>>> print danceablility
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
NameError: name 'danceablility' is not defined
>>> print danceability
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: __str__ returned non-string (type NoneType)
>>> print %r %  danceability
  File "<stdin>", line 1
    print %r %  danceability
          ^
SyntaxError: invalid syntax
>>> print "%r" %  danceability
<essentia.standard.Algo object at 0x7f8c2591ca70>
>>> print "%s" %  danceability
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: __str__ returned non-string (type NoneType)
>>> reader = essentia.standard.MetadataReader(filename = /home/bzachmann/Documents/hello_brooklyn.mp3)
  File "<stdin>", line 1
    reader = essentia.standard.MetadataReader(filename = /home/bzachmann/Documents/hello_brooklyn.mp3)
                                                         ^
SyntaxError: invalid syntax
>>> reader = essentia.standard.MetadataReader(filename = '/home/bzachmann/Documents/hellobrooklyn.mp3')
>>> metadata = reader()
>>> print reader.title
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: 'Algo' object has no attribute 'title'
>>> print reader
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: __str__ returned non-string (type NoneType)
>>> print "%s" %  reader.title
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: 'Algo' object has no attribute 'title'
>>> print "%s" % reader.title
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: 'Algo' object has no attribute 'title'
>>> print "%s" % reader
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: __str__ returned non-string (type NoneType)
>>> print "%r" % reader
<essentia.standard.Algo object at 0x7f8c2581f4d0>
>>> print "%s" % metadata.title
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: 'tuple' object has no attribute 'title'
>>> print metadata.title
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: 'tuple' object has no attribute 'title'
>>> print metadata.Title
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: 'tuple' object has no attribute 'Title'
>>> print metadata.artist
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: 'tuple' object has no attribute 'artist'
>>> test =  metadata.title
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: 'tuple' object has no attribute 'title'
>>> print "%r" % metadata.title
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: 'tuple' object has no attribute 'title'
>>> print "%r" % metadata
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: not all arguments converted during string formatting
>>> print "%s" % metadata
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: not all arguments converted during string formatting
>>> print "%r" % metadata
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: not all arguments converted during string formatting
>>> print "%r" % metadata.title
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: 'tuple' object has no attribute 'title'
>>> print "%r" % metadata.chanels
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: 'tuple' object has no attribute 'chanels'
>>> print "%r" % metadata.channels
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: 'tuple' object has no attribute 'channels'
>>> print "%r" % metadata.title()
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: 'tuple' object has no attribute 'title'
>>> print "%r" % metadata.tags
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: 'tuple' object has no attribute 'tags'
>>> print "%r" % metadata.tag
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: 'tuple' object has no attribute 'tag'
>>> print "%r" % metadata.title()
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: 'tuple' object has no attribute 'title'
>>> print "%r" % metadata.tags.title
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: 'tuple' object has no attribute 'tags'
>>> print "%r" % metadata.tags()
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: 'tuple' object has no attribute 'tags'
>>> metadata
('Hello, Brooklyn', 'All Time Low', 'Nothing Personal', '', '', '7/12', '2009', <essentia.Pool object at 0x7f8c39d3d180>, 206, 64, 22050, 2)
>>> metadata(1)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: 'tuple' object is not callable
>>> metadata[1]
'All Time Low'
>>> metadata[1]
'All Time Low'
>>> print "%s" metadata[1]
  File "<stdin>", line 1
    print "%s" metadata[1]
                      ^
SyntaxError: invalid syntax
>>> print "%s" %  metadata[1]
All Time Low

